file_documentation/weekplanner/blocs/toolbar_bloc.md

```diff
@@ -0,0 +1,10 @@
+# Documentation of toolbar_bloc.dart
+This file documents the `toolbar_bloc.dart` file from the weekplanner project.
+
+## Functionality
+`toolbar_bloc.dart` is responsible for emitting icons through the stream `visibleButtons` which is used by the Giraf appbar to determine which icons it should display.
```

Comment:
Isn't it also responsible for dialogs?

Reply:
Yes, some of the functionality of the icon buttons are to show a dialog. I added a little at the end of the Code section.